charts/prerequisites/values.yaml

```diff
@@ -65,6 +65,34 @@ postgresql:
     # For better security, add postgresql-secrets k8s secret with postgres-password, replication-password and password
     existingSecret: postgresql-secrets
 
+gcloud-sqlproxy:
+  enabled: false
+  ## Specify an existing secret holding the cloud-sql service account credentials
+  existingSecret: ""
+  ## The key in the existing secret that stores the credentials
+  existingSecretKey: ""
+  ## SQL connection settings
+  ##
+  cloudsql:
+    ## PostgreSQL/MySQL instances:
+    ## update with your GCP project, the region of your Cloud SQL instance
+    ## and the name of your Cloud SQL instance
+    ## PostgreSQL port 5432 or MySQL port 3306, or other port you set for your SQL instance.
+    ## Use different ports for different instances.
+    instances:
+      # GCP instance name.
+    - instance: "instance"
```

Comment:
Remove all these placeholder values. Just leave them empty.
The comments should make it clear what needs to be filled in.